hooks/pre/10-reset_feature.rb

```diff
@@ -18,6 +18,12 @@ def foreman_installed?
   $?.success?
 end
 
+response = ask('Are you sure you want to continue? [y/n]')
+unless response == 'y'
+  $stderr.puts '** cancelled **'
+  kafo.class.exit(1)
+end
```

Comment:
This gets asked on every invocation of the installer even if you didn't specify `--reset`.

It's also in a bit of an awkward place since it's in the middle of a bunch of method defintions.  It should go to the bottom.